internal/driver/interactive.go

```diff
@@ -42,6 +42,16 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 	interactiveMode = true
 	shortcuts := profileShortcuts(p)
 
+	// Get all groups in pprofVariables to allow for clearer error messages.
+	groups := make(map[string][]string)
```

Comment:
It might be better to make this a separate function like
```
func (v *variable) groupOptions() map[string][]string {
```
in commands.go, as this `interactive()` function got a bit big. Probably also push the sorting down as well so that this function returns sorted list of option names in each named group.

Reply:
Done.